### PR TITLE
Create crowdstart from suggestions

### PR DESCRIPTION
**Pull request by xkjyeah**

### CODE REVIEW

File: src/lib/models/Bid.js [hunk 1]
```diff
@@ -42,7 +45,49 @@ export default function (modelCache) {
     indexes: [
       {fields: ["userId"]},
       {fields: ["routeId"]}
-    ]
+    ],
+
+    classMethods: {
+      async createForUserAndRoute (userInst, routeInst, price, options = {}) {
```
Comment (LoneRifle) [inline src/lib/models/Bid.js]: Matter to consider: `models.Bid.createForUserAndRoute` or `models.Bid.forUserAndRoute`? The former lines up better with Sequelize naming conventions, while the latter is somewhat more readable, and clearer that this is not a Sequelize in-built

File: scripts/setup_virtualenv.sh [hunk 2]
```diff
@@ -2,6 +2,7 @@
 # this script is meant to be run using `pg_virtualenv`
 
 export DATABASE_URL=postgres://$PGUSER:$PGPASSWORD@$PGHOST:$PGPORT/$PGDATABASE
+export BEELINE_ROUTING_SERVER=https://routing.beeline.sg
```
Comment (LoneRifle) [inline scripts/setup_virtualenv.sh]: `ROUTING_URL` perhaps?

File: src/lib/models/Bid.js (hunk 1 above)
Comment (xkjyeah) [reply src/lib/models/Bid.js]: erm... but user and route are most certainly not sequelize built-ins. I don't see how anyone can be confused by this.

Comment (LoneRifle) [reply src/lib/models/Bid.js]: Not so much confusion as it is a matter of style. As noted, is fine anyway.

**Comment by LoneRifle** [comment]
Can you rebase your changes so that you can pick up some of the lint fixes I've done earlier?

**Pull request merged**